filetests/isa/intel/binary64.cton

```diff
@@ -145,6 +145,149 @@ ebb0:
+    ; asm: movswl (%rsi), %edx
+    [-,%rdx]            v405 = sload16.i32 v2   ; bin: 40 0f bf 16
+    ; asm: movzbl (%rcx), %edi
+    [-,%rdi]            v406 = uload8.i32 v1    ; bin: 40 0f b6 39
+    ; asm: movzbl (%rsi), %edx
+    [-,%rdx]            v407 = uload8.i32 v2    ; bin: 40 0f b6 16
+    ; asm: movsbl (%rcx), %edi
+    [-,%rdi]            v408 = sload8.i32 v1    ; bin: 40 0f be 39
+    ; asm: movsbl (%rsi), %edx
+    [-,%rdx]            v409 = sload8.i32 v2    ; bin: 40 0f be 16
+
+    ; Register-indirect with 8-bit signed displacement.
+
+    ; asm: movq %rcx, 100(%rsi)
+    store v1, v2+100                            ; bin: 48 89 4e 64
+    ; asm: movq %rsi, -100(%rcx)
+    store v2, v1-100                            ; bin: 48 89 71 9c
+    ; asm: movl %ecx, 100(%rsi)
+    istore32 v1, v2+100                         ; bin: 40 89 4e 64
+    ; asm: movl %esi, -100(%rcx)
+    istore32 v2, v1-100                         ; bin: 40 89 71 9c
+    ; asm: movw %cx, 100(%rsi)
+    istore16 v1, v2+100                         ; bin: 66 40 89 4e 64
+    ; asm: movw %si, -100(%rcx)
+    istore16 v2, v1-100                         ; bin: 66 40 89 71 9c
+    ; asm: movb %cl, 100(%rsi)
+    istore8 v1, v2+100                          ; bin: 40 88 4e 64
+    ; asm: movb %sil, 100(%rcx)
+    istore8 v2, v1+100                          ; bin: 40 88 71 64
+
+    ; asm: movq 50(%rcx), %rdi
+    [-,%rdi]            v462 = load.i64 v1+50           ; bin: 48 8b 79 32
+    ; asm: movq -50(%rsi), %rdx
+    [-,%rdx]            v463 = load.i64 v2-50           ; bin: 48 8b 56 ce
+    ; asm: movl 50(%rcx), %edi
+    [-,%rdi]            v464 = uload32.i64 v1+50        ; bin: 40 8b 79 32
+    ; asm: movl -50(%rsi), %edx
+    [-,%rdx]            v465 = uload32.i64 v2-50        ; bin: 40 8b 56 ce
+    ; asm: movslq 50(%rcx), %rdi
+    [-,%rdi]            v410 = sload32.i64 v1+50        ; bin: 48 63 79 32
+    ; asm: movslq -50(%rsi), %rdx
+    [-,%rdx]            v411 = sload32.i64 v2-50        ; bin: 48 63 56 ce
+    ; asm: movzwq 50(%rcx), %rdi
+    [-,%rdi]            v412 = uload16.i64 v1+50        ; bin: 48 0f b7 79 32
+    ; asm: movzwq -50(%rsi), %rdx
+    [-,%rdx]            v413 = uload16.i64 v2-50        ; bin: 48 0f b7 56 ce
+    ; asm: movswq 50(%rcx), %rdi
+    [-,%rdi]            v414 = sload16.i64 v1+50        ; bin: 48 0f bf 79 32
+    ; asm: movswq -50(%rsi), %rdx
+    [-,%rdx]            v415 = sload16.i64 v2-50        ; bin: 48 0f bf 56 ce
+    ; asm: movzbq 50(%rcx), %rdi
+    [-,%rdi]            v416 = uload8.i64 v1+50         ; bin: 48 0f b6 79 32
+    ; asm: movzbq -50(%rsi), %rdx
+    [-,%rdx]            v417 = uload8.i64 v2-50         ; bin: 48 0f b6 56 ce
+    ; asm: movsbq 50(%rcx), %rdi
+    [-,%rdi]            v418 = sload8.i64 v1+50         ; bin: 48 0f be 79 32
+    ; asm: movsbq -50(%rsi), %rdx
+    [-,%rdx]            v419 = sload8.i64 v2-50         ; bin: 48 0f be 56 ce
```

Comment:
Exactly, this is the right set of instructions to test here. You should do the same for the no-displacement ones above.